app/src/constants/userConstants.ts

```diff
@@ -0,0 +1,4 @@
+export enum UserRole {
+  ADMIN = "ADMIN",
+  MEMBER = "MEMBER",
+}
```

Comment:
I always debate whether or not enums for this kinda thing is worth the effort. I know we do it a fair amount - non-blocking but when something can be expressed as string literal types very simply it begs the question a bit

Reply:
i'll pull from the server - then we can get rid of this I think